arrow-libs/core/arrow-meta/src/main/java/arrow/meta/encoder/jvm/TypeElementEncoder.kt

```diff
@@ -430,6 +430,7 @@ interface TypeElementEncoder : KotlinMetatadataEncoder, KotlinPoetEncoder, Proce
       Modifier.SYNCHRONIZED -> null
       Modifier.NATIVE -> null
       Modifier.STRICTFP -> null
+      else -> null
```

Comment:
This could be handled move gracefully depending on the version but it's out of the scope of this ticket. Happy to add a new one to look at ways to support newer modifiers.
cc: @raulraja

Reply:
This is regarding SEALED and NON\_SEALED modifiers introduced in JDK 15

Reply:
whatever works, most of this code in the old meta part is going away soon and the new meta does not depend on the TypeeElement java apis.